models/tmp/stg_intercom__team_admin_tmp.sql

```diff
@@ -1,5 +1,5 @@
---To disable this model, set the using_team_admin variable within your dbt_project.yml file to False.
-{{ config(enabled=var('using_team', True)) }}
+--To disable this model, set the intercom__using_team_admin variable within your dbt_project.yml file to False.
```

Comment:
same comment as before -- `intercom__using_team_admin` or `intercom__using_team`?

Reply:
team_admin was removed during dev and must have snuck its way in. Removed, thanks!